util/util.go

```diff
@@ -148,3 +148,11 @@ func RemoveOrphanedInstances(verbose bool) error {
 
 	return nil
 }
+
+// AbsTarPathMatches tells whether the tar header name matches the path.
+// This function is needed since some tar files prefix its header names
+// with / and some not. NOTE: argument 'path' should start with '/' while
+// 'tarPath' should be header.Name.
+func AbsTarPathMatches(path, tarPath string) bool {
+	return tarPath == path || "/"+tarPath == path
```

Comment:
Hm, the `filepath.Abs` seems to be too smart to be useful in this case:
```
// If the path is not absolute it will be joined with the current
// working directory to turn it into an absolute path.
```
So in my case `filepath.Abs("meta/run.yaml")` yields `/home/miha/PLAYGROUND/apache-spark/meta/run.yaml`. Not sure how it works on GoPlay - perhaps each example is run inside `/` of its own Docker Container.